R/fb_ad.R

```diff
@@ -183,7 +183,7 @@ fbad_list_ad <- function(fbacc, id, statuses, fields = 'id') {
     }
 
     ## default ID for current Ad Account
-    if (missing(id) | fn == 'fbad_list_campaign') {
+    if (missing(id) | fn %in% c('fbad_list_campaign', 'fbad_list_audience')) {
```

Comment:
Audiences can be shared across accounts but you're right that they exist *on* an account and appropriate to require initializing to an account to get its audiences.